repose-aggregator/src/docs/asciidoc/architecture/servlet-spec.adoc

```diff
@@ -0,0 +1,22 @@
+= Servlet Specification
+
+The https://javaee.github.io/servlet-spec/[Java Servlet Specification] is a document that details the design and API of a Java-based system that can respond to requests.
+To learn more about the specification itself, visit the link near the start of this paragraph.
+
+*Repose* is built on top of the servlet specification.
+However, *Repose* also constrains the specification in certain ways.
+For example, *Repose* can only service HTTP requests.
+
+To understand the architecture of *Repose*, it is vital that one understands the servlet specification.
+Currently, *Repose* complies with version 3.1 of the specification.
```

Comment:
Pretty sure it's actually 3.0, we have a couple of classes that don't implement the added methods of 3.1.

Reply:
Ah, I think you are right. Will update.